web/packages/test/scripts/deploy-ethereum.sh

```diff
@@ -76,6 +87,10 @@ deploy_local()
     echo "Waiting for geth API to be ready"
     sleep 3
 
+    if [ "$eth_fast_mode" == "true" ]; then
+      hack_beacon_client
+    fi
```

Comment:
Seems in previous commit we remove `fast_mode`, would prefer we add it back since it's useful to quick test something without waiting for a long time, meanwhile it's just an option not enabled by default.